org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserProjectFileManager.java

```diff
@@ -0,0 +1,15 @@
+package org.code.javabuilder;
+
+public class UserProjectFileManager implements ProjectFileManager {
+
+  public UserProjectFileManager(String baseUrl, String[] fileNames) {}
+
+  public void loadFiles() throws UserFacingException {
+    throw new UserFacingException("We couldn't load your files");
```

Comment:
put a todo here?

Reply:
I'll leave the TODO off this time as it's implemented in this follow-up PR: https://github.com/code-dot-org/java-ide/pull/14